src/Sarif.Sarifer/BackgroundAnalysisService.cs

```diff
@@ -17,37 +18,93 @@ namespace Microsoft.CodeAnalysis.Sarif.Sarifer
     [Export(typeof(IBackgroundAnalysisService))]
     internal class BackgroundAnalysisService : IBackgroundAnalysisService
     {
-#pragma warning disable CS0649 // Filled in by MEF
-#pragma warning disable IDE0044 // Assigned by MEF
+#pragma warning disable IDE0044, CS0649 // Provided by MEF
         [ImportMany]
         private IEnumerable<IBackgroundAnalyzer> analyzers;
-#pragma warning restore IDE0044
-#pragma warning restore CS0649
+
+        [ImportMany]
+        private IEnumerable<IBackgroundAnalysisSink> sinks;
```

Comment:
>sinks [](start = 53, length = 5)

You know, I really like this. It always bothered me a little that the service imported the analyzers, but the analyzer (base) imported the sinks. This is much better.